pkg/driver/version.go

```diff
@@ -78,5 +79,12 @@ func UserAgent(kubernetesVersion string) string {
 		b.WriteString(kubernetesVersion)
 	}
 
+	if authenticationSource != "" {
```

Comment:
Authentication source should never be empty, so it's probably fine to remove this